n/numba/numba_0.62.0dev0_ubi_9.3.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : numba
+# Version       : 0.62.0dev0
+# Source repo   : https://github.com/numba/numba
+# Tested on     : UBI:9.3
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Haritha Nagothu <haritha.nagothu2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on the given
+# platform using the mentioned version of the package.
+# It may not work as expected with newer versions of the
+# package and/or distribution. In such a case, please
+# contact the "Maintainer" of this script.
+#
+# -----------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=numba
+PACKAGE_VERSION=${1:-0.62.0dev0}
+PACKAGE_URL=https://github.com/numba/numba
+PACKAGE_DIR=numba
+SCRIPT_DIR=$(pwd)
+
+# Install dependencies and tools.
+
+dnf install -y --nodocs https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+yum install -y git gcc gcc-c++  make wget python3.11 python3.11-devel python3.11-pip xz-devel bzip2-devel openssl-devel zlib-devel libffi-devel llvm15-devel.ppc64le
+
+export LLVM_CONFIG=/usr/lib64/llvm15/bin/llvm-config
+export CFLAGS=-I/usr/include
+export CXXFLAGS=-I/usr/include
+
+#clone repository
+git clone $PACKAGE_URL
+cd  $PACKAGE_DIR
+git checkout $PACKAGE_VERSION
+
+python3.11 -m pip install numpy setuptools
```

Comment:
Pls check if numpy needs to be pinned to v2.0.2

In OpenCE we are building with numpy v2.0.2.